tempesta_fw/cache.c

```diff
@@ -487,7 +488,8 @@ node_db(void)
 static int
 tfw_cache_sched_cpu(TfwHttpReq *req)
 {
-	return c_nodes[req->node].cpu;
+	struct ca_node_t *node = &ca_nodes[req->node];
+	return node->cpu[node->cpu_idx++ % node->nr_cpus];
```

Comment:
If many cores update the same cache line, then there is still huge amount of memory transfers due to frequent invalidation of the cache line. If performance is critical, then the counter should be per_cpu. However, I suppose this isn't a hot spot.